### DISCUSSION TITLE
The otter.nvim plugin no longer requires otter-nvim-cmp

### DISCUSSION THREAD

**Discussion by refaelsh** [nix-community/nixvim#2243] [Q&A] [answered]
The plugin has the following line:
https://github.com/nix-community/nixvim/blob/2d5949f65ca6060552ca5841716119c6d7d4bb54/plugins/by-name/otter/default.nix#L22

But, cmp integration no longer required, as stated in their README right at the top:
```
If you previously used the otter nvim-cmp source, you can remove it, as the completion results now come directly via the cmp-nvim-lsp source together with other language servers.
```

**Comment by refaelsh** [comment]
CC @PerchunPak.

**Comment by PerchunPak** [comment]
Hi, I am now using nixCats instead of nixvim, so cannot help you unfortunately

**Reply by refaelsh** [reply]
> Hi, I am now using nixCats instead of nixvim, so cannot help you unfortunately

Sure. I understand.

In that case: @MattSturgeon, would that be ok if I open a PR to remove the `otter-nvim-cmp` from the plugin's options?

**Reply by khaneliman** [reply, marked as answer]
> > Hi, I am now using nixCats instead of nixvim, so cannot help you unfortunately
> 
> Sure. I understand.
> 
> In that case: @MattSturgeon, would that be ok if I open a PR to remove the `otter-nvim-cmp` from the plugin's options?

Always feel free to open PRs for solutions you have found. We can discuss implementation details and merit within that easily enough.

**Reply by refaelsh** [reply]
Then I will try and open a PR for that. Thank you :-)
